CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # SwissPayment Changelog
 
+## 1.1.0 (2021-xx-yy)
+
+  * Renamed Mixed to MixedMoney (reserved word as of PHP 7).
+  * Dropped support for PHP 5.6.
```

Comment:
`Dropped support for PHP 5.6.`
belongs to v1.0

1.1 would need a notice that support for PHP 7.0 to 7.1 was dropped.